R/stack.R

```diff
@@ -507,6 +509,14 @@ getVarInEnv <- function(name, env) {
   })
 }
 
+getActiveBinding <- function(name, env){
+  ret <- list(
```

Comment:
We have to condition on the R version:
```r
ret <- if (getRversion() >= "4.0.0") {
  activeBindingFunction(name, env)
} else {
  "The R version must be at least 4.0.0 to get the function of an active binding"
}
structure(list(bindingFunction = ret), class = '.csv.ActiveBinding')
```

Reply:
Thanks! I went with the version suggested by @renkun-ken , but in case this doesn't work in some version, I left a (commented) line in the function to return an error message.